### PR TITLE
Techjobs console readme

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+1. Implement the 'printJobs' method
+2. Create the `findByValue` method
+3. Make search methods `Case-Insensitive`
+
+## Unit Tests 
+
+The starter code contains unit tests and text files. There is one test class that contains 4 tests. These tests compare students' output to the text files.  
+
+We encourage students to run unit tests after they complete the assignment before submitting. Directions for how to run tests have been provided in the [Java](https://education.launchcode.org/java-web-development/) textbook's [Assignment 0](https://education.launchcode.org/csharp-web-development/assignments/hello-world.html).
```
Comment (codinglikeagirl42) [inline README.md]: The first link is going to the Java Book not C#

File: README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+   Technical Assistant / User Support
+   Technical Writer
+   Web - Back End
+   Web - Front End
+   Web - Full Stack
+   ```   
+
+## Submitting Your Work
+
+To submit your work please follow the [Submission Instructions](https://education.launchcode.org/charp-web-development/assignments/hello-world.html#submitting-your-work).
```
Comment (codinglikeagirl42) [inline README.md]: Link does not work.

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

File: README.md (hunk 1 above)
Comment (gildedgardenia) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (gildedgardenia) [reply README.md]: Fixed!

**Review by codinglikeagirl42 — APPROVED** [review]

**Pull request merged**